src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -129,6 +140,12 @@
           + " Use 0 for no timeout";
   public static final String SIMPLE_CONSUMER_POOL_TIMEOUT_MS_DEFAULT = "1000";
 
+  public static final String SIMPLE_CONSUMER_MAX_POLL_TIME_CONFIG = "simpleconsumer.max.poll.time";
```

Comment:
Do we want to continue using the "simple consumer" nomenclature? These setting is user-facing and I think might be the only user-facing place we mention the simple consumer. The reason I am wondering is that the simple/high-level naming is based on the old consumers and has always been pretty confusing for users as the names are really from the perspective of Kafka developers (SimpleConsumer is anything but simple for users).

More generally, will these just end up being normal consumer settings as well when we convert the rest of the code to the new consumer? In which case maybe we can just prefix them with `consumer`? Or come up with some other naming which more clearly explains the difference between the manual partition assignment and group-based consumers?

/cc @hachikuji